example_workspace/crazy.hlasm

```diff
@@ -1,15 +1,15 @@
-* MACRO CYCLES
-* it is possible to recursively call macro from itself
-* the number of calls is limited in HLASM 
-    MACRO
-    MAC
-    MAC
-    MEND 
+***************************  INFINIT CYCLES  **************************
+* It is possible to recursively call macro from itself.
+* The number of calls is limited in HLASM:
+         MACRO
+         MAC
+         MAC
+         MEND 
 
-    MAC
-    
+         MAC
 
-    AGO .SKIPRUBBISH
-    Anything in here is just skipped.
-    And it is still valid hlasm source code.
-.SKIPRUBBISH ANOP
+**************************  RUBBISH SKIPPING  *************************
```

Comment:
At this point we can call it a block comment, right?

Reply:
Agreed. I've merged this with comments.hlasm and renamed crazy.hlasm to just infinite.hlasm